atlas-fastq-provider-functions.sh

```diff
@@ -138,6 +149,29 @@ validate_ena_ssh_path() {
     fi
 }
 
+
+# Use AWS S3 to check if the file is in ENA. Note that the '-n' is important,
+# because any 'while read' loops calling this script use STDIN, which SSH will
+# consume otherwise.
+
+validate_ena_fire_path() {
+    local enaFile=$1
+    
+    check_ena_s3_profile
+
+    if [ $? -eq 1 ]; then
+        return 1
+    fi 
+
+    aws --profile $ENA_S3_PROFILE --endpoint-url $ENA_S3_URL s3 ls $enaFile > /dev/null 2>&1
```

Comment:
Dont you need to check if `$ENA_S3_URL` exists?

Reply:
Ah good point.  It isn't checked because it's supposedly in the default config.  But this reminds me to add it and other new variables there.  Thanks!